plugin/ochttp/server.go

```diff
@@ -140,6 +141,11 @@ type trackingResponseWriter struct {
 }
 
 var _ http.ResponseWriter = (*trackingResponseWriter)(nil)
+var _ http.Hijacker = (*trackingResponseWriter)(nil)
+
+func (t *trackingResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
```

Comment:
Shouldn't we do it only if the underlying writer is a http.Hijacker?

Reply:
If the question is why we have a `Hijack` method, it is because we can't dynamically create an `http.Hijack` method at runtime for a type. But yes, if the underlying writer is an http.Hijacker, invoking Hijacker() works fine, and if it is on an http2 connection, it panics as the sample code even without OpenCensus shows.

Reply:
Thanks for the advocacy and discussions offline. We came to the consensus that  if the underlying responseWriter doesn't implement http.Hijacker, return an error, like other implementations do. I've updated that code and tests, PTAL.